### ISSUE TITLE
Accessibility: Unlabeled Buttons

### ISSUE THREAD

**Issue by Menelion** [Cryde/php-vote#14]
I'm a screen reader user, and I find your website quite accessible, but there are some unlabeled buttons. The most problems are in the editor toolbar, but I believe it's an external dependency from another author, isn't it?  
Anyway, I would like to label some buttons and probably extend WAI-ARIA support, and I'm creating this issue just to know if those changes are welcome. If they are, I'll send a PR shortly.  
And one more thing: there is an empty `translations` directory. Is i18n planned? If so, should I wrap my labels in some function call or just add them in plain English? Thanks!

**Comment by Cryde** [comment]
Hello !

> I'm a screen reader user, and I find your website quite accessible, but there are some unlabeled buttons. The most problems are in the editor toolbar, but I believe it's an external dependency from another author, isn't it?

Yes indeed it's an external dep !

> Anyway, I would like to label some buttons and probably extend WAI-ARIA support, and I'm creating this issue just to know if those changes are welcome. If they are, I'll send a PR shortly.

Any PR are welcome ! :D 

> And one more thing: there is an empty translations directory. Is i18n planned? If so, should I wrap my labels in some function call or just add them in plain English? Thanks!

For now just use plain English, I will prepare translatioin later (the only problem I see with translation is that "ideas" will be writen in one language - so only the interface will be translated (or maybe let the users translate idea - don't know yet))

Thank you a lot
